recipes/spritz/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.3.3" %}
+
+package:
+  name: spritz
+  version: {{ version }}
+
+source:
+  url: https://github.com/smith-chem-wisc/Spritz/releases/download/{{ version }}/SpritzCMD.zip
+  sha256: f218e3ebb599b29960ae0245aa4a16ab8734ed7895d98f0772b1f7734dacfc10
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - sysroot_linux-64 2.17  # [linux64]
+  run:
+    - snakemake-minimal =5.27.4
+    - mamba
+    - python=3.8
```

Comment:
any reason why you pin python and dotnet?

Reply:
Spritz's commandline tool uses dotnet 5.0, and it doesn't work with other frameworks/runtimes, so that needs to be pinned.

Reply:
It needs python 3. Should I just remove the pin, or pin it to `python=3`?

Reply:
We do not build for python2 anymore. So you can really remove that.

Reply:
I'm giving it a try with these pins removed, other than for `dotnet-runtime`.